### PR TITLE
refactor (*): remove templates 🚧

### PR DESCRIPTION
**Pull request by fsamin**
don't merge

### CODE REVIEW

**Comment by sguiheux** [comment]
Don't forget UI Impact => create an empty application call Void template

**Comment by fsamin** [comment]
indeed

File: ui/package-lock.json [hunk 1]
```diff
@@ -1635,6 +1635,7 @@
       "requires": {
         "anymatch": "1.3.2",
         "async-each": "1.0.1",
+        "fsevents": "1.1.3",
```
Comment (yesnault) [inline ui/package-lock.json]: there a link with templates?

Comment (fsamin) [reply ui/package-lock.json]: i don't think so but every npm install i ran this last week end added this

**Pull request merged**